config/database.yml

```diff
@@ -23,16 +23,16 @@ default: &default
 
 development:
   <<: *default
-  database: Recipes_development
+  database: Recipe_app_development
 
   # The specified database role being used to connect to postgres.
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: Recipes
+  username: postgres
 
   # The password associated with the postgres role (username).
-  #password:
+  password: 0805Snsd01
```

Comment:
Hello @MahmoudBakr23 , on windows it requires me to put a password on that file otherwise I can't open the live server. What can I do to fix that issue?